docs/reference/client-script-usage.md

```diff
@@ -1,17 +1,22 @@
-Using the Sigrid CI client script
-=================================
+Using the Sigrid CI client scripts
+==================================
 
-Sigrid CI consists of a number of Python-based client scripts, that interact with Sigrid in order to analyze your project's source code and provide feedback based on the results. These client scripts need to be available to the CI environment, in order to call the scripts *from* the CI pipeline. The [general Sigrid CI documentation](../README.md) contain instructions on how to make this script available within various CI environments. 
+Sigrid CI consists of a number of Python-based client scripts, that interact with Sigrid to support two different kinds of development process integration:
 
-Once the `sigridci.py` script is available within your CI environment, you can call the script to start the Sigrid CI run. 
+- To *publish* your code to Sigrid after every change.
```

Comment:
Here's a proper suggestion you can apply (I noticed that in my earlier comment, I had put the closing double quote at the wrong place):
```suggestion
- To publish your code to Sigrid after every change. Publishing makes results available at https://sigrid-says.com for users who have access to the system.
```